### ISSUE TITLE
CRAN compliance: Test Packages "baseballstats" and "sartre" should not write to \Library folder

### ISSUE THREAD

**Issue by bburns632** [UptakeOpenSource/pkgnet#83]
`pkgnet` has a minor CRAN compliance issue.  I believe this is due to both issue #78 and our custom subpages for testing and vignettes being built in the \Library folder.  The [log files on CRAN](https://cran.r-project.org/web/checks/check_results_pkgnet.html) seem to support this theory.

>   Packages should not write in the user’s home filespace (including
>   clipboards), nor anywhere else on the file system apart from the R
>   session’s temporary directory (or during installation in the location
>   pointed to by TMPDIR: and such usage should be cleaned up).

Modifications or alternates to the `devtools::install_local` functions should be explored in the unit tests and the vignette.

**Comment by jayqi** [comment]
Regarding the `devtools::install_local` issue, what we can do is instead use `devtools::load_all` to temporarily load the package into memory. This means we'll need to change the checks for `installed.packages` to take the union with any loaded package namespaces. We can capture those with:
```
grep('^package:', search(), value = TRUE)
```

**Comment by jayqi** [comment]
Update on the above approach: unfortunately this works for `FunctionReporter` but not for  `DependencyReporter`. Code below:

```
# devtools::install_local('inst/baseballstats')
remove.packages('baseballstats')

devtools::load_all('inst/baseballstats')

# In FunctionReporter, works fine
numFuncs <- as.character(unlist(utils::lsf.str(asNamespace('baseballstats')))) # list of functions within Package
numFuncs

# In FunctionReporter, works fine
funcMap <- suppressWarnings({
    mvbutils::foodweb(
        where = paste("package"
                      , 'baseballstats'
                      , sep = ":")
        ,
        plotting = FALSE
    )
})
funcMap

# In DependencyReporter, always returns null 
# Should return:
# $baseballstats
# [1] "base"    "methods" "utils" 
db <- utils::installed.packages()
tools::package_dependencies('baseballstats'
                            , reverse = FALSE
                            , recursive = FALSE
                            , db = db
                            , which = "Imports"
)

# Another function in tools, returns character(0)
tools::dependsOnPkgs('baseballstats'
                     , recursive = FALSE
                     , dependencies = "Imports"
)
```

**Comment by jameslamb** [comment]
This is fixed as of #108
